sdk/anomalydetector/Azure.AI.AnomalyDetector/tests/samples/Sample1_DetectEntireSeriesAnomaly.cs

```diff
@@ -57,26 +57,38 @@ public async Task DetectEntireSeriesAnomaly()
             //detect
             Console.WriteLine("Detecting anomalies in the entire time series.");
 
-            EntireDetectResponse result = await client.DetectEntireSeriesAsync(request).ConfigureAwait(false);
-
-            if (result.IsAnomaly.Contains(true))
+            try
             {
-                Console.WriteLine("An anomaly was detected at index:");
-                for (int i = 0; i < request.Series.Count; ++i)
+                EntireDetectResponse result = await client.DetectEntireSeriesAsync(request).ConfigureAwait(false);
+
+                if (result.IsAnomaly.Contains(true))
```

Comment:
This syntax is odd, since `IsAnomaly` reads as if it's a single boolean rather than a set.   I'd recommend a comment above to clarify your intent of "if any one of these is true, than the entire thing is considered an anomaly."

Reply:
Changed the logic here: Iterate through the IsAnomaly list, if there is anomaly, print "An anomaly was detected at index: [index]." and if there is not any anomaly detected, print "No anomalies detected in the series."